service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatus.java

```diff
@@ -35,7 +35,7 @@
     private String conversationId;
     private EhrRequest ehrRequest;
     private GpcAccessStructured gpcAccessStructured;
-    private List<GpcAccessDocument> gpcAccessDocuments;
+    private GpcAccessDocument gpcAccessDocument;
```

Comment:
Do we need a double nesting?

Reply:
yes it is required

Reply:
Ok

Reply:
there are a couple of access document endpoints we need to call. For example the output of "find patient" will be under gpcAccessDocument.patient